bin/init-aio.sh

```diff
@@ -43,13 +43,26 @@ set -a
 if [ "$is_start_http" = "true" ]; then
     . /config/http.env
     /usr/bin/neofs-http-gw &
+
+    while [[ "$(curl -s -o /dev/null -w %{http_code} localhost:8081)" != "404" ]];
+    do
+      sleep 1;
+    done
 fi
 
 if [ "$is_start_rest" = "true" ]; then
     . /config/rest.env
     /usr/bin/neofs-rest-gw &
+
+    while [[ "$(curl -s -o /dev/null -w %{http_code} localhost:8090)" != "404" ]];
+    do
+      sleep 1;
+    done
 fi
 
+./bin/tick.sh
```

Comment:
Why a tick here? All of the settings were configured above and gateways don't care much about these ticks.

Reply:
This tick is not connected to the gates. It is connected to the NeoFS. As I remember, you can't upload objects until epoch number 3. You just receive an error like "no nodes/hosts to select from". This tick solves this issue

Reply:
I checked one more time the tests in all gates with the new AIO version without this tick. Also, I removed the tick inside the tests. The result is green, and all tests passed. It is a good news. Removed this row from sh file